src/systems/ec/implementations/vanilla/packages/ec-component-table/ec-component-table.scss

```diff
@@ -162,6 +162,47 @@
       padding-top: $ecl-spacing-s;
     }
   }
+
+  // SortTable
+  .ecl-table__header[data-ecl-table-sort-toggle] {
+    cursor: pointer;
+    padding-left: calc(
+      #{$ecl-spacing-m} + #{$ecl-icon-2-xs} + #{$ecl-spacing-xs}
+    );
+    position: relative;
+  }
+
+  .ecl-table__arrow {
+    left: $ecl-spacing-m;
+    position: absolute;
+  }
+
+  .ecl-table__icon-up {
+    fill: $ecl-color-grey-50;
+    left: 0;
+    position: absolute;
```

Comment:
These are repeated throughout both up and down elements, a candidate for `.ecl-table__icon` if you ask me

Reply:
done